articles/compliance/gdpr/features-aiding-compliance/user-consent.md

```diff
@@ -0,0 +1,353 @@
+
+Sample request:
+
+```har
+{
+    "method": "GET",
+    "url": "https://${account.namespace}/api/v2/users-by-email",
+    "httpVersion": "HTTP/1.1",
+    "headers": [{
+        "name": "Authorization",
+        "value": "Bearer YOUR_MGMT_API_ACCESS_TOKEN"
+    }],
+    "queryString":  [
+        {
+          "name": "email",
+          "value": "USER_EMAIL_ADDRESS"
+        },
+        {
+          "name": "fields",
+          "value": "user_metadata"
+        }
+    ]
+}
+```
+
+Sample response:
+
+```json
+[
+  {},
+  {
+    "user_metadata": {
+      "consent": {
+	    "given": true,
+	    "date": "01/23/2018",
+	    "text_details": "some-url"
+	  }
+    }
+  }
+]
+```
+
+### Search for a user using their Id
+
+To search for a user using their Id, use [the Get a user endpoint](/users/search#users-by-id). 
+
+Set the **fields** request parameter to `user_metadata` in order to limit the fields returned. This way, only the `user_metadata` will be returned instead of the complete user profile.
+
+Sample request:
+
+```har
+{
+    "method": "GET",
+    "url": "https://${account.namespace}/api/v2/users/YOUR_USER_ID",
+    "httpVersion": "HTTP/1.1",
+    "headers": [{
+        "name": "Authorization",
+        "value": "Bearer YOUR_MGMT_API_ACCESS_TOKEN"
+    }],
+    "queryString":  [
+        {
+          "name": "fields",
+          "value": "user_metadata"
+        }
+    ]
+}
+```
+
+Sample response:
+
+```json
+{
+  "user_metadata": {
+    "consent": {
+	    "given": true,
+	    "date": "01/23/2018",
+	    "text_details": "some-url"
+  	}
+  }
+}
+```
+
+### Search for a set of users
+
+To search for a set of users, use [the List or search users endpoint](/users/search#users). 
+
+This endpoint is eventually consistent (that is, the response might not reflect the results of a recently-complete write operation) and that [only specific fields are available for search](/api/management/v2/user-search#searchable-fields). 
+
+Consent information that are saved as part of the `user_metadata` are not searchable. 
+
+For a sample request and response see [Search Users](/users/search#users). For more examples, see [Example Queries](/api/management/v2/user-search#example-queries).
+
+### Update consent information
+
+To update a user's `user_metadata`, use [the Update a user endpoint](/api/management/v2#!/Users/patch_users_by_id).
+
+How you will structure your request depends on how you have structured your metadata: as root or as inner properties.
+
+Metadata as root properties:
+
+```json
+{
+  "consentGiven": true,
+  "consentDetails": "some-url"
+}
+```
+
+Metadata as inner properties:
+
+```json
+{
+  "consent": {
+    "given": true,
+    "text_details": "some-url"
+  }
+}
+```
+
+#### Update a root property
+
+In this case the metadata are merged so you need only send the field you want to update. For example, let's say we want to add a consent date and set it to `01/23/2018`.
```

Comment:
Updates to root-level properties are merged, so you only need to send the value for the field you wanted updated.